projects/angular/src/popover/signpost/_signposts.clarity.scss

```diff
@@ -222,14 +266,13 @@
           signpost-variables.$clr-signpost-pointer-border,
           variables.$clr-use-custom-properties
         );
-
-        top: -1 * signpost-variables.$clr-signpost-border-size + variables.$clr_baselineRem_1px;
+        top: -1 * signpost-variables.$clr-signpost-border-size;
 
         &:before {
           @include mixins.css-var(
             border-bottom,
-            clr-signpost-pointer-psuedo-border,
-            signpost-variables.$clr-signpost-pointer-psuedo-border,
```

Comment:
This is a breaking rename to an existing CSS custom property and SCSS variable. Since it's would break products even without the shim being applied, we should not include this change in a non-major release. I would file an issue to rename this in a breaking change PR for v16.